### ISSUE TITLE
Stuck on startup

### ISSUE THREAD

**Issue by wolfrik17** [noggaholic/machinima-studio#17]
When I start the program it just stays on Updating Offsets and never does anything after that

**Comment by karliky** [comment]
@wolfrik17 Are you using the latest release? https://github.com/noggaholic/machinima-studio/releases

Is GW2 running in 32 bit mode?

**Comment by wolfrik17** [comment]
Latest release and running GW2 with the -32 argument and no other addons installed

**Comment by karliky** [comment]
Have you tried to run it as admin?

**Comment by wolfrik17** [comment]
Running it as admin does nothing. I also tried version 2.0.0 but that one gets stuck on Checking Dependencies.

**Comment by karliky** [comment]
I've published a new release https://github.com/noggaholic/machinima-studio/releases/tag/2.0.2

This new release includes a link upon start up that says "Click to open dev tools.".

This will open the developer tools window, go to the "console" tab and see if there are any errors there

**Comment by wolfrik17** [comment]
According to the console error GW2 is running in 64 bit mode instead of 32 bit despite me using the -32 argument when launching the game. I'm guessing the -32 argument isn't working for some reason.

I tried downloading the 32 bit game client from the website but it looks like it's installing the 64 bit version anyway. 

Downloading the 32 bit setup, renaming it to Gw2.exe, making a shortcut of this file and running the -32 argument on this new file made it run in 32 bit and the program works without issues now.

**Comment by karliky** [comment]
You should definetly try to find a way to run GW2 in 32 mode.

I've attached both of my gw2-*.exe here, maybe if you place them in the folder you can get it working with the -32 argument
[Guild Wars 2.zip](https://github.com/noggaholic/machinima-studio/files/3512548/Guild.Wars.2.zip)

**Comment by wolfrik17** [comment]
I think you missed the edit I did to my last message. 

The way I got it to work in 32 bit was to download the 32 bit Gw2Setup from the website, rename it to Gw2.exe, make a shortcut from Gw2.exe (originally the 32 bit Gw2Setup) and use the -32 argument on this shortcut. Using the -32 argument on the shortcut I would make from Gw2-64.exe wouldn't work. 

The program works fine now thanks for the support.
